lib/sinon/stub-non-function-property.js

```diff
@@ -3,15 +3,13 @@
 var valueToString = require("./util/core/value-to-string");
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
-function stubNonFunctionProperty(object, property, value) {
+function stubNonFunctionProperty(object, property) {
```

Comment:
I have pushed another commit to improve documentation for stubbing out non-function properties